modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/delivery/ConcurrentContentReadTaskBatchProcessor.java

```diff
@@ -33,11 +33,12 @@
 import org.wso2.andes.kernel.ProtocolMessage;
 
 import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Concurrently read content read tasks and batch the event and forward the batched events to event handler
+ * Concurrently read content read tasks and batch the deliveryEventArrayList and forward the batched events to deliveryEventArrayList handler
```

Comment:
Incorrect wording. I think better to use event instead of deliveryEventArrayList.